nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/cmd/exec/internal/search/DefaultSearchFunctionHandlerLookup.java

```diff
@@ -151,6 +158,13 @@ private R sortIfApplicable(List<ParamValue> paramValues, ModelConfig<?> mConfig,
 		if(index == -1)
 			throw new IllegalStateException("Invalid orderby clause for StaticCodeValue search for a command "+cmd+" . It must follow the pattern \"property.direction\" e.g. code.asc() or name.firstName.desc()");
 		
+		if(inMemorySortLimit <= 0 || paramValues.size() > inMemorySortLimit) {
+			logit.error(() ->"Size of paramValues "+paramValues.size()+" is greater than Configured limit for in memory sort: "+inMemorySortLimit+", so returning the list without sorting. "
```

Comment:
This is good. Could you also include a flag to indicate if an exception should be thrown instead of just logging and returning unsorted value? I see the value of returning at least the original value which can be left as default behavior, but worst and in rare case, the user can override it and fail fast.

Reply:
Added the check, please review